tests/app/models/DevicesModelTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Models\DevicesModel;
+
+class DeicesModelTest extends BaseModelTest
+{
+    public function testConstruct()
+    {
+        $deviceModel = new DevicesModel($this->getConnection());
+    }
+
+    public function testAdd()
+    {
+        $_POST = array('Name' => 'dummy', 'Description' => 'lorem ipsum', 'Type' => 1);
+
+        $deviceModel = new DevicesModel($this->getConnection());
```

Comment:
`setUp()` should give you a fresh instance before each test is run.  `setUp()` is automatically called before each individual test is run and is a good place to put code that is shared among many tests.

https://phpunit.de/manual/current/en/fixtures.html